metadata-ingestion/src/datahub/ingestion/source/snowflake/snowflake_lineage.py

```diff
@@ -299,58 +363,71 @@ def _get_upstream_lineage_info(
                 self.report.upstream_lineage[dataset_name] = [
                     u.dataset for u in upstream_tables
                 ]
-            return (
-                UpstreamLineage(
-                    upstreams=upstream_tables,
-                    fineGrainedLineages=sorted(
-                        finegrained_lineages, key=lambda x: (x.downstreams, x.upstreams)
-                    )
-                    or None,
-                ),
-                column_lineage,
+            return UpstreamLineage(
+                upstreams=upstream_tables,
+                fineGrainedLineages=sorted(
+                    finegrained_lineages, key=lambda x: (x.downstreams, x.upstreams)
+                )
+                or None,
             )
+
         return None
 
     def _populate_view_lineage(self, conn: SnowflakeConnection) -> None:
         with PerfTimer() as timer:
             self._populate_view_upstream_lineage(conn)
             self.report.view_upstream_lineage_query_secs = timer.elapsed_seconds()
-        with PerfTimer() as timer:
-            self._populate_view_downstream_lineage(conn)
-            self.report.view_downstream_lineage_query_secs = timer.elapsed_seconds()
+        if self.report.edition == SnowflakeEdition.STANDARD:
+            logger.info(
+                "Snowflake Account is Standard Edition. View to Table Lineage Feature is not supported."
```

Comment:
just a question - do we need to return here? why continue?

Reply:
reason mentioned above.